src/Apps/Artist/Server/redirect.ts

```diff
@@ -0,0 +1,18 @@
+export function redirectWithCanonicalParams({ req, res }) {
+  if (hasCanonicalSort(req.query)) {
+    // all good
+    return
+  }
+
+  // else remove the offending sort, causing fallback to default sort
+  delete req.query.sort
+
+  const sanitizedQuerystring = new URLSearchParams(req.query).toString()
```

Comment:
**nitpick**: rename to `sanitizedQueryString`

Reply:
Heh, I was this was consistent out there, I followed [the Node convention](https://nodejs.org/api/querystring.html) of not camel casing this 🤷🏽?

Reply:
TIL 😅